src/website/views/custom-elements/date-picker.pug

```diff
@@ -0,0 +1,67 @@
+---
+title: Date Picker
+scripts:
+  - js/ce/ux-chi-ce
+styles:
+  - assets/styles/datepicker-ce
+---
+p.-text
+  | Date pickers are used for inputting dates from the user. It might be toggled when the user focuses
+  | an input of type date in a form.
+
+h3 Examples
+p.-text
+  | Here there are some examples of calendars not attached to any element.
+
+ul#a-tabs--dp.a-tabs
+  li.-active
+    a(href='#a-tabs--dp-base') Base
+  li
+    a(href='#a-tabs--dp-active') Active
+  li
+    a(href='#a-tabs--dp-min-max') Min & max
+#a-tabs--dp-base.a-tabs-panel.-active
+  .example.-mb--3(style='overflow:hidden;')
+    .-p--3
+      chi-date
+    :code(lang="html")
+      <chi-date></chi-date>
+#a-tabs--dp-active.a-tabs-panel
+  .example.-mb--3(style='overflow:hidden;')
+    .-p--3
+      chi-date(value="05/15/2019")
+    :code(lang="html")
+      <chi-date value="05/15/2019"></chi-date>
+#a-tabs--dp-min-max.a-tabs-panel
+  .example.-mb--3(style='overflow:hidden;')
+    .-p--3
+      chi-date(min="06/06/2019", max="06/22/2019")
+    :code(lang="html")
+      <chi-date min="06/06/2019" max="06/22/2019"></chi-date>
+
+h3 Input field
+p.-text
+  | Use <code>chi-date-picker</code> for including a date-picker input.
+.example.-mb--3
+  .-p--3
+    div(style="max-width: 14rem;")
+      chi-date-picker
+  :code(lang="html",style="height:20rem")
+    <chi-date-picker></chi-date-picker>
+
+h4 Value, min and max options
+.example.-mb--3
+  .-p--3
+    div(style="max-width: 14rem;")
+      chi-date-picker(value="01/24/2019", min="01/14/2019", max="02/07/2019")
+  :code(lang="html",style="height:20rem")
+      <chi-date-picker value="01/24/2019" min="01/14/2019" max="02/07/2019">
+script.
+  document.addEventListener(
+    'DOMContentLoaded',
+    function () {
+      // TODO Migrate tabs to Custom Elements.
```

Comment:
I think this is not necessary

Reply:
It is not possible right now, so I'll remove the comment. Thank you!